README.md

```diff
@@ -7,70 +7,81 @@ SCALE is a light-weight format which allows encoding (and decoding) which makes
+structures look) needs to be known separately at both encoding and decoding ends.
 The encoded data does not include this contextual information.
 
 To get a better understanding of how the encoding is done for different types,
 take a look at the
-[low-level data formats overview page at the Substrate docs site](https://docs.substrate.io/reference/scale-codec/).
+[SCALE Code page at the Substrate Knowledge Base](https://docs.substrate.io/v3/advanced/scale-codec/).
 
 ## Implementation
 
 The codec is implemented using the following traits:
 
 ### Encode
 
-The `Encode` trait is used for encoding of data into the SCALE format. The `Encode` trait contains the following functions:
+The `Encode` trait is used for encoding of data into the SCALE format. The `Encode` trait
+contains the following functions:
+
 * `size_hint(&self) -> usize`: Gets the capacity (in bytes) required for the encoded data.
 This is to avoid double-allocation of memory needed for the encoding.
 It can be an estimate and does not need to be an exact number.
-If the size is not known, even no good maximum, then we can skip this function from the trait implementation.
-This is required to be a cheap operation, so should not involve iterations etc.
-* `encode_to<T: Output>(&self, dest: &mut T)`: Encodes the value and appends it to a destination buffer.
+If the size is not known, even no good maximum, then we can skip this function from the trait
+implementation. This is required to be a cheap operation, so should not involve iterations etc.
+* `encode_to<T: Output>(&self, dest: &mut T)`: Encodes the value and appends it to a destination
+  buffer.
 * `encode(&self) -> Vec<u8>`: Encodes the type data and returns a slice.
-* `using_encoded<R, F: FnOnce(&[u8]) -> R>(&self, f: F) -> R`: Encodes the type data and executes a closure on the encoded value.
+* `using_encoded<R, F: FnOnce(&[u8]) -> R>(&self, f: F) -> R`: Encodes the type data and
+  executes a closure on the encoded value.
 Returns the result from the executed closure.
 
-**Note:** Implementations should override `using_encoded` for value types and `encode_to` for allocating types.
-`size_hint` should be implemented for all types, wherever possible. Wrapper types should override all methods.
+**Note:** Implementations should override `using_encoded` for value types and `encode_to` for
+allocating types. `size_hint` should be implemented for all types, wherever possible. Wrapper
+types should override all methods.
 
 ### Decode
 
-The `Decode` trait is used for deserialization/decoding of encoded data into the respective types.
+The `Decode` trait is used for deserialization/decoding of encoded data into the respective
+types.
 
-* `fn decode<I: Input>(value: &mut I) -> Result<Self, Error>`: Tries to decode the value from SCALE format to the type it is called on.
+* `fn decode<I: Input>(value: &mut I) -> Result<Self, Error>`: Tries to decode the value from
+  SCALE format to the type it is called on.
```

Comment:
Double spaces

Reply:
Hm, that's actually what we want to indicate that the text's part of the list's bullet point. :P I'll reformat it.